application/src/main/java/net/corda/v5/application/interop/InteropIdentityLookUp.java

```diff
@@ -0,0 +1,32 @@
+package net.corda.v5.application.interop;
+
+import net.corda.v5.application.interop.facade.FacadeId;
+import net.corda.v5.base.annotations.DoNotImplement;
+import net.corda.v5.base.annotations.Suspendable;
+import net.corda.v5.interop.InterOpIdentityInfo;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+/**
+ * <p>This interface represents the injectable service for client corDapp to look up the remote alias identity.
```

Comment:
We would need to clearly separate this service from any services returning normal member group, i would use explicit words like _Interoperability group_ etc. Also javadoc for other services doesn't mention `injectable`, all Corda API services are injectable.
Maybe something like that, which explicitly shows the different with existing `memberLookup`:
``The service allows a flow to retrieve alter-ego members from other networks, which are register in the holding identity's interoperability group.``